File: PetshopDBApi/PetshopDBApi/wwwroot/Scripts/Models/Product.js
class Product extends Entity {
    constructor(json)
    {
        super(json);
        if (json) {
            this.Category = json.category;
            this.Title = json.title;
            this.Brand = json.brand;
            this.Price = json.price;
            this.Quantity = json.quantity;
        }
        else
        {
            this.Category = "";
            this.Title = "";
            this.Brand = "";
            this.Price = 0;
            this.Quantity = 0;
        }
    }
}

File: PetshopDBApi/PetshopDBApi/Lib/Models/Product.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace PetshopDBApi
{
    public class Product : Entity
    {
        public string Category { get; set; }
        public string Title { get; set; }
        public string Brand { get; set; }
        public double Price { get; set; }
        public int Quantity { get; set; }
    }
}
